ehrql/backends/tpp.py

```diff
@@ -296,6 +296,32 @@ def modify_query_variables(self, variables):
         """
     )
 
+    covid_therapeutics = QueryTable(
+        """
+        SELECT DISTINCT
+            Patient_ID AS patient_id,
+            COVID_indication AS covid_indication,
+            Count AS count,
+            CurrentStatus AS current_status,
+            Diagnosis AS diagnosis,
+            FormName AS form_name,
+            Intervention AS intervention,
+            CASIM05_date_of_symptom_onset,
+            MOL1_onset_of_symptoms,
+            SOT02_onset_of_symptoms,
+            STUFF(coalesce(',' + NULLIF(REPLACE(REPLACE(REPLACE(CASIM05_risk_cohort, 'Patients with a ', ''),  'Patients with ', ''), ' and ', ','), ''), ''), 1, 1, '') as CASIM05_risk_cohort,
+            STUFF(coalesce(',' + NULLIF(REPLACE(REPLACE(REPLACE(MOL1_high_risk_cohort, 'Patients with a ', ''),  'Patients with ', ''), ' and ', ','), ''), ''), 1, 1, '') as MOL1_high_risk_cohort,
+            STUFF(coalesce(',' + NULLIF(REPLACE(REPLACE(REPLACE(SOT02_risk_cohorts, 'Patients with a ', ''),  'Patients with ', ''), ' and ', ','), ''), ''), 1, 1, '') as SOT02_risk_cohorts,
+            STUFF(coalesce(',' + NULLIF(MOL1_high_risk_cohort, ''), '') + coalesce(',' + NULLIF(SOT02_risk_cohorts, ''), '') + coalesce(',' + NULLIF(CASIM05_risk_cohort, ''), ''), 1, 1, '') as risk_cohort,
```

Comment:
Does this also handle the filler word replacing?

Reply:
good spot - it actually didn't do what I thought it was doing - the final aggregation column was using the raw columns not the cleaned columns (they were selected as the same name, which misled me). Now fixed.

Reply:
(and tested!)